conf/content_host.yaml.template

```
@@ -2,51 +2,51 @@ content_host:
   default_rhel_version: 7
   rhel6:
     vm:
-      workflow: deploy-base-rhel
+      workflow: deploy-rhel
       deploy_rhel_version: '6'
       target_memory: 1GiB
       target_cores: 1
     container:
       container_host: rhel6:latest
   rhel7:
     vm:
-      workflow: deploy-base-rhel
+      workflow: deploy-rhel
       deploy_rhel_version: '7'
       target_memory: 1GiB
       target_cores: 1
     container:
       container_host: ubi7:latest
   rhel7_fips:
     vm:
-      workflow: deploy-base-rhel-fips
+      workflow: deploy-rhel
       deploy_rhel_version: '7'
       target_memory: 1GiB
       target_cores: 1
   rhel8:
     vm:
-      workflow: deploy-base-rhel
+      workflow: deploy-rhel
       deploy_rhel_version: '8'
       target_memory: 1536 MiB
       target_cores: 1
     container:
       container_host: ubi8:latest
   rhel8_fips:
     vm:
-      workflow: deploy-base-rhel-fips
+      workflow: deploy-rhel
       deploy_rhel_version: '8'
       target_memory: 1536 MiB
       target_cores: 1
   rhel9:
     vm:
-      workflow: deploy-base-rhel
+      workflow: deploy-rhel
       deploy_rhel_version: '9'
       target_memory: 1536 MiB
       target_cores: 1
     container:
       container_host: ubi9:latest
   rhel9_fips:
     vm:
-      workflow: deploy-base-rhel-fips
+      workflow: deploy-rhel
```

@jyejare , no matter what, don't you need to add an entry for RHEL 10 and the correct workflow call there?

Please note that if implemented this way, the value for `workflow` will need to be changed on GA and the new non-GA will have to be inserted afterwards.

```suggestion
      workflow: deploy-template
      deploy_rhel_version: '10'
```

---

Yes it needs update for rhel10.  The main change is in CI as the actual conf resides there, this is just a template.

---

Addressed by adding a new config for RHEL10.